src/App.tsx

```diff
@@ -23,6 +23,7 @@ function App() {
 
           <Route path="tarifler" element={<Tariffs />} />
           <Route path="Mağazalar" element={<Stores />} />
+          <Route path="Məxfilik" element={<Confidentiality />} />
```

Comment:
"path" is something that you type here. so I think you need to change your paths to be more easier to write (because I can write Mağazalar without copy-paste 😂). you can do it in some other task I think)
![image](https://user-images.githubusercontent.com/18329725/206645628-bf0f2eef-1b1d-4fde-8da9-5fd0f814b626.png)